datasets/dave/dataset.json

```diff
@@ -0,0 +1,119 @@
+{
+  "id": "dave",
+  "type": "dataset",
+  "auth": "OPENBAAR",
+  "authorizationGrantor": "n.v.t.",
+  "theme": [
+    "Test, Dogfood & Demo"
+  ],
+  "homepage": "https://data.amsterdam.nl",
+  "owner": "Gemeente Amsterdam, Data Verwerking",
+  "spatialDescription": "Amsterdam",
+  "version": "0.0.1",
+  "title": "Dave",
+  "language": "nl",
+  "dateCreated": "2023-08-28T14:00:00+01:00",
+  "license": "Creative Commons, Naamsvermelding",
+  "accrualPeriodicity": "dagelijks",
+  "creator": "Datateam Dataverwerkingen",
+  "publisher": {
+    "$ref": "publishers/DAVE"
+  },
+  "description": "Test dataset bedoeld als integratie test.",
+  "status": "beschikbaar",
+  "keywords": [
+    "dave",
+    "2001SpaceOdyssey",
+    "test",
+    "demo",
+    "dogfood",
+    "dogfooding"
+  ],
+  "crs": "EPSG:28992",
+  "objective": "Developers kunnen end-to-end keten testen",
+  "temporalUnit": "",
+  "tables": [
+    {
+      "id": "pedigree",
+      "type": "table",
+      "version": "0.0.1",
+      "title": "Afvalwijzer Amsterdam",
+      "description": "Table 1 voor de end-to-end test.",
+      "schema": {
+        "$schema": "http://json-schema.org/draft-07/schema#",
+        "type": "object",
+        "additionalProperties": false,
+        "required": [
+          "schema",
+          "id"
+        ],
+        "mainGeometry": "geometrie",
+        "display": "id",
+        "properties": {
+          "schema": {
+            "$ref": "https://schemas.data.amsterdam.nl/schema@v1.1.1#/definitions/schema"
+          },
+          "key": {
+            "type": "integer",
+            "description": "Primary Key."
+          },
+          "id": {
+            "type": "integer",
+            "description": "Uniek id in order to test SCD."
+          },
+          "active": {
+            "type": "boolean",
+            "description": "Test boolean values, set to TRUE for active FALSE for inactive."
+          },
+          "dateIngest": {
+            "type": "string",
+            "format": "date-time",
+            "description": "Timestamp of ingestion into this table UTC."
+          },
+          "dateGenerated": {
+            "type": "string",
+            "format": "date-time",
+            "description": "Timestamp of data generation UTC."
+          },
+          "comment": {
+            "type": "string",
+            "description": "Comment field."
+          },
+          "picture": {
+            "type": "bytea",
```

Comment:
Deze gaat niet lukken. Ik had je ff de link met de spec moeten mailen. Hierbij alsnog: https://schemas.data.amsterdam.nl/docs/ams-schema-spec.html#data-types

Je ziet dit ook terug in de validatie.

Reply:
Ah, yes, check; dwz dat er geen manier is om binary data te uploaden?

Reply:
Ik heb een string field met base64 encoding toegevoegd, denk dat dit voldoende moet zijn